resources/solr-configuration/src/main/resources/patients/conf/schema.xml

```diff
@@ -61,6 +61,7 @@
     <field name="reporter" type="string" indexed="false" stored="true" required="true" />
     <field name="visibility" type="string" indexed="true" stored="true" required="false" />
     <field name="accessLevel" type="int" indexed="true" stored="true" required="false" />
+    <field name="phenotype_ancestors" type="string" indexed="true" stored="true" multiValued="true"/>
```

Comment:
Similarly we have extended_negative_phenotype and extended_negative_prenatal_phenotype. In that perspective do we need a new solr field, negative_phenotype_ancestors?